src/artemis/external_interaction/tests/test_zocalo_handler.py

```diff
@@ -0,0 +1,65 @@
+from unittest.mock import MagicMock, call
+
+from artemis.external_interaction.communicator_callbacks import FGSCallbackCollection
+from artemis.parameters import FullParameters
+from artemis.utils import Point3D
+
+
+def test_run_gridscan_zocalo_calls(
```

Comment:
Nit: Can we change the name of this test to be a bit more descriptive?

Reply:
sure, I didn't name it :p